File: app/src/main/java/com/dgd/earthquakes/models/IQuake.java
package com.dgd.earthquakes.models;

import android.location.Location;

import java.util.Date;

/**
 * Created by Max
 * on 22-Jun-17.
 */

public interface IQuake {

    Date getDate();

    void setDate(Date mDate);

    Location getLocation();

    void setLocation(Location mLocation);

    String getMagnitude();

    void setMagnitude(String mMagnitude);

    String getLink();

    void setLink(String mLink);

    String getId();

    void setId(String mId);

    String getTitle();

    void setTitle(String mTitle);
}

File: app/src/main/java/com/dgd/earthquakes/models/IQuake.kt
package com.dgd.earthquakes.models

import java.util.*

/**
 * Created by Max
 * on 22-Jun-17.
 */

interface IQuake {

    var date: Date?

    var latitude: Double?

    var longitude: Double?

    var magnitude: String?

    var link: String?

    var id: String?

    var title: String?
}
